pkg/broker/broker_test.go

```diff
@@ -4,17 +4,31 @@ import (
 	"context"
 
 	. "github.com/cloudfoundry-incubator/blockhead/pkg/broker"
+	"github.com/cloudfoundry-incubator/blockhead/pkg/config"
```

Comment:
fix this. dot import or import normally, not both.

Reply:
derp, 
read too quickly, 
fine as is. 
somewhat concerned by the dot import at all tho.